### DISCUSSION TITLE
Relational field that are readonly should not show a placeholder neither v (for choice)

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15542] [Feature Requests]
Relational field that are readonly show "Select an item..." and the v for choice.
They should only show as an empty readonly input.

(Use case: only an admin has editing access to this field which the user can see)

### To Reproduce

Create an M2O field in a collection with condition:
```
{
	"_and": [
		{
			"user_created": {
				"_eq": "$CURRENT_USER"
			}
		}
	]
}
```

`Disable editing value`

The regular user see `Select an item...`  and `v` (which is not relevant as s.he cannot edit the item)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
